dcpy/lifecycle/ingest/run.py

```diff
@@ -1,10 +1,19 @@
 import typer
 
-from . import extract, transform
+from dcpy.connectors.edm import recipes
+from . import configure, extract, transform, TMP_DIR
 
 
 def run(dataset: str, version: str | None = None):
-    config = extract.extract_and_archive_raw_dataset(dataset, version)
+    config = configure.get_config(dataset, version)
+
+    # download dataset
+    extract.download_file_from_source(
+        config.source, config.raw_filename, config.version
```

Comment:
Should probably clean this up a bit, a little silly to be passing three subfields of the same object